conans/client/graph/spdx.py

```diff
@@ -0,0 +1,58 @@
+
+def generate_sbom(conanfile, **kwargs):
+    name = conanfile.ref.name
+    version = conanfile.ref.version
+    date = datetime.fromtimestamp(time.time(), tz=timezone.utc).strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
Which date should be here? Maybe the revision date is more correct?

Reply:
the creation date identify when the SPDX document was originally created

https://spdx.github.io/spdx-spec/v2-draft/document-creation-information/#69-created-field